tests/util/kube_utils.go

```diff
@@ -90,8 +90,8 @@ func CreateAndFill(outDir, templateFile string, values interface{}) (string, err
 }
 
 // CreateNamespace create a kubernetes namespace
-func CreateNamespace(n string) error {
-	if _, err := ShellMuteOutput("kubectl create namespace %s", n); err != nil {
+func CreateNamespace(n string, kubeconfig string) error {
```

Comment:
I haven't studied the e2e code in detail, so YMMV on my commentary.    Something feels dissatisfying about passing every value of the struct AppManager, rather than just passing AppManager and using directly.  Perhaps this is to abstract that structure, although the abstraction falls apart as every value is passed in every test.

Reply:
These utils are used in lots of different e2e files many of them don't deal with an AppManager structure at all so changing as you suggest would be quite involved.

Reply:
as I mentioned I had not studied the e2e code all that much.